tests/SimpleStringTest.cpp

```diff
@@ -405,7 +411,7 @@ TEST(SimpleString, findNormal)
     SimpleString str("Hello World");
     LONGS_EQUAL(0, str.find('H'));
     LONGS_EQUAL(1, str.find('e'));
-    LONGS_EQUAL(-1, str.find('!'));
+    LONGS_EQUAL(SimpleString::npos, str.find('!'));
```

Comment:
In my mind, this is a lot cleaner than returning an int where it should be size_t, and then comparing to -1. I find the "int approach" is old-school c-ish, not c++.